pkg/roachprod/roachprod.go

```diff
@@ -659,7 +660,42 @@ func Start(
 	if err != nil {
 		return err
 	}
-	return c.Start(ctx, l, startOpts)
+	if err := c.Start(ctx, l, startOpts); err != nil {
+		return err
+	}
+
+	if startOpts.ScheduleBackups {
+		if err := createBackupSchedule(context.Background(), l, c.Name, c.IsLocal(), c.Secure); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+// createBackupSchedule creates a cluster backup schedule which runs an incremental every 15
+// minutes and a full every hour
+func createBackupSchedule(
+	ctx context.Context, l *logger.Logger, clusterName string, isLocal bool, secure bool,
+) error {
+	externalStoragePath := `gs://cockroachdb-backup-testing`
+
```

Comment:
@srosenberg the bucket above has been used for nightly roachtests, though I'm wondering if a different bucket, perhaps with a different gc policy, should be used for roachprod clusters.

Reply:
Indeed, this bucket has a TTL of 1 day,

```
Delete object | 1+ days since object was updated
```

The above bucket is in `cockroach-shared` whereas roachprod uses `cockroach-ephemeral` by default. We should probably create a separate bucket with a different retainment policy. How is this envisioned to be used? Manual testing only?